sass/elements/_kbd.scss

```diff
@@ -0,0 +1,12 @@
+kbd {
+  font-size: 0.8rem;
+  padding-left: 0.1rem;
+  padding-right: 0.1rem;
+  padding-top: 0.1rem;
+  padding-bottom: 0.1rem;
+  line-height: 1;
+  border-color: grey;
+  border-radius: 0.3rem;
+  border-style: solid;
+  border-width: 1px;
```

Comment:
Bikeshed: I kind of like how GitHub styles their `kbd` elements (seen in the issue). It makes it look kinda like a key or button. Would adding a larger bottom width look good?

Reply:
<kbd>test</kbd> it actually uses `box-shadow: inset 0 -1px 0 var(--borderColor-neutral-muted, var(--color-neutral-muted));`

Reply:
So, what's about it? As for me, the flat style looks more clear. Plus I don't remember any shadow on the site, even assets used there, I mean the rendered one, are flat too.